java_servlets/src/main/java/checkServlet.java

```diff
@@ -0,0 +1,21 @@
+import com.google.gson.Gson;
+
+import javax.servlet.annotation.WebServlet;
+import javax.servlet.http.HttpServlet;
+import javax.servlet.http.HttpServletRequest;
+import javax.servlet.http.HttpServletResponse;
+import java.io.IOException;
+import java.io.PrintWriter;
+
+@WebServlet("/check")
```

Comment:
Hope you understand that servlets are defined either by annotations or through web.xml. Afraid I haven't stressed it on during the lecture. There's no need to define both ways